pkgs/applications/audio/sublime-music/default.nix

```diff
@@ -38,12 +40,21 @@ python3Packages.buildPythonApplication rec {
     "python-mpv"
   ];
 
+  postPatch = ''
+    sed -i "/--cov/d" setup.cfg
+    sed -i "/--no-cov-on-fail/d" setup.cfg
+    # https://github.com/sublime-music/sublime-music/pull/370
+    # Can be removed in later versions (probably > 0.11.16)
+    substituteInPlace pyproject.toml \
+      --replace 'python-Levenshtein = "^0.12.0"' 'Levenshtein = ">0.12.0"'
```

Comment:
How about fetchpatching from https://github.com/sublime-music/sublime-music/commit/6c6f7ec72f83d643af8638f9380e6227a01809c5 ?

Reply:
Also the project has migrated to GitHub.

Reply:
I migrated from poetry to flit in addition to moving the project to GitHub, so the patch would not work.

I need to figure out what to do about a couple of failing tests and then I'll just make a release and I'll do an overhaul of the module at that point.